tribe/views.py

```diff
@@ -54,6 +54,22 @@ def get_queryset(self):
         return queryset
 
 
+class ArticleTextViewSet(
+    ListModelMixin,
+    UpdateModelMixin,
+    GenericViewSet
+):
+    serializer_class = ArticleTextSerializer
+    queryset = ArticleText.objects.all()
+
+    def get_queryset(self):
+        queryset = ArticleText.objects.all()
+        article = self.request.query_params.get('articleid')
```

Comment:
A bit nitpicky but I think important for the consistency, change this to 'article'. We use 'tribe' and not 'tribeid' for the other filters too.

Reply:
Changed it. I had to also change the model for it.